### PR TITLE
created cards component

### PR DESCRIPTION
**Pull request by sagargse**
Created card component

### CODE REVIEW

**Review by Rockyraven — APPROVED** [review]
all good for me

File: cssfiles/cards.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
+.all-cards{
+    display: flex;
+    gap:2rem;
+    margin:1rem;
+}
+.vertical-card{
+    display: flex;
+    flex-direction: column;
+    width:20rem;
+    background-color: var(--card-color);
```
Comment (dikshadhote) [inline cssfiles/cards.css]: use color name instead of card color

File: cssfiles/cards.css [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    border-radius: 0.5rem;
```
Comment (dikshadhote) [inline cssfiles/cards.css]: use px for values below 10px

File: cssfiles/cards.css [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    height:fit-content;
+    position: relative;
+}
+.vertical-card>img{
+    border-top-left-radius: 0.5rem;
+    border-top-right-radius: 0.5rem;
```
Comment (dikshadhote) [inline cssfiles/cards.css]: same as mentioned above

**Review by pratik-kumar-sahu — COMMENTED** [review]
- For "out of stock" card you can disable add to Cart & move to Wishlist buttons

File: cssfiles/cards.css [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+}
+.card-desc-block{
+    margin: 0.5rem;
```
Comment (dikshadhote) [inline cssfiles/cards.css]: same as above

File: cssfiles/cards.css [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+}
+.cart-button{
+    background-color: var(--primary);
+    padding: 0.5rem;
+    border-radius: 0.5rem;
```
Comment (dikshadhote) [inline cssfiles/cards.css]: same as above

File: cssfiles/cards.css [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+    color: var(--white-color);
+}
+.cart-button:hover{
+    background-color: var(--btn-hover);
```
Comment (dikshadhote) [inline cssfiles/cards.css:29]: use color names, btn-hover doesn't gives idea about color you are using

File: cssfiles/cards.css [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+}
+.wish-button{
+    border: 3px solid var(--primary);
+    padding: 0.5rem;
+    border-radius: 0.5rem;
```
Comment (dikshadhote) [inline cssfiles/cards.css]: same as above

File: cssfiles/cards.css [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+}
+.wish-button:hover{
+    background-color: var(--btn-hover);
```
Comment (dikshadhote) [inline cssfiles/cards.css:37]: same as above

File: cssfiles/cards.css [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+    color: var(--white-color)
+}
+.card-buttons{
+    display: flex;
+    gap:1rem;
+    border-radius: 0.5rem;
```
Comment (dikshadhote) [inline cssfiles/cards.css]: same as above

File: cssfiles/cards.css [hunk 10]
```diff
@@ -0,0 +1,70 @@
...
+    position: absolute;
+    color: var(--white-color);
+    bottom: 4rem;
+    left:1rem;
+}
+.horizontal-card{
+    display: flex;
+    width:fit-content;
+    background-color: var(--card-color);
+    border-radius: 0.5rem;
```
Comment (dikshadhote) [inline cssfiles/cards.css]: same as above

File: cssfiles/cards.css [hunk 11]
```diff
@@ -0,0 +1,70 @@
...
+    height:10rem;
+    position: relative;
+}
+.horizontal-card>img{
+    border-top-left-radius: 0.5rem;
+    border-bottom-left-radius: 0.5rem;
```
Comment (dikshadhote) [inline cssfiles/cards.css]: same as above

File: cssfiles/cards.css [hunk 12]
```diff
@@ -0,0 +1,70 @@
...
+.horizontal-card>img{
+    border-top-left-radius: 0.5rem;
+    border-bottom-left-radius: 0.5rem;    
+}
+.text-body{
+    position: absolute;
+    top: 4rem;
+    left:2rem;
+    background-color: var(--card-color);
+    padding: 0.5rem;
```
Comment (dikshadhote) [inline cssfiles/cards.css]: same as above

File: main.css [hunk 13]
```diff
@@ -22,6 +22,7 @@
     --secondary-alert:#70809075;
     --light-gray:#00000079;
     --lightish:#ff000000;
+    --card-color:#d3d3d3ee;
```
Comment (dikshadhote) [inline main.css]: use color names as you have used for light gray

**Review by dikshadhote — CHANGES_REQUESTED** [review]
The code looks good , just use proper naming convention and px for values below 10px as suggested by vikram.

**Comment by sagargse** [comment]
> * For "out of stock" card you can disable add to Cart & move to Wishlist buttons

ok i will do that

File: main.css (hunk 13 above)
Comment (sagargse) [reply main.css]: okay will correct that

File: cssfiles/cards.css (hunk 2 above)
Comment (sagargse) [reply cssfiles/cards.css]: okay!

File: cssfiles/cards.css (hunk 1 above)
Comment (sagargse) [reply cssfiles/cards.css]: okay!

**Pull request merged**